qiskit/providers/ibmq/api/clients/schema.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
Non blocking, and mostly a note for follow-ups (as I think the decision had to do with the circular import quick fix) - for organizational purposes, we should find a different place for this file, keeping the `clients/` module just for clients.

Reply:
Yes it was moved here to resolve circular import issue, as it's used by `AccountClient`. I'll open a follow up issue to address this.